architecture/users/readme.md

```diff
@@ -90,9 +84,9 @@
     ]
     ```
 
-### USR-F3 Я как пользователь хочу иметь возможность получить пользователей по их slack name.
+### USR-F3 I, as a user, want to be able to get list of students on their slack name.
 
-* USR-F3-D1 Входные данные - массив slackNames, результат - массив users с полями uuid, slack, skype и name.
+* USR-F3-D1 At the input there is a slackNames array, at the output there is an array of students with the uuid, slack, skype and name fields.
```

Comment:
~At the input there is~ **Input -** a slackNames array, ~at the~ output **-** ~there is~ an array of students with the uuid, slack, skype and name fields.